src/managers/world-manager.js

```diff
@@ -3,35 +3,34 @@
 *
 * As discussed under the [World API Adapter](../world-api-adapter/), a [run](../../../glossary/#run) is a collection of end user interactions with a project and its model. For building multiplayer simulations you typically want multiple end users to share the same set of interactions, and work within a common state. Epicenter allows you to create "worlds" to handle such cases.
 *
-* The [World API Adapter](../world-api-adapter/) handles creating multiplayer worlds, and adding and removing end users and runs from a world.
-*
-* The World Manager provides an easy way to track and access the current world and run for particular end users.
+* The World Manager provides an easy way to track and access the current world and run for particular end users. It is typically used in code for end user -facing pages in your project. (The related [World API Adapter](../world-api-adapter/) handles creating multiplayer worlds, and adding and removing end users and runs from a world. Because of this, typically the World Adapter is used in code for facilitator pages in your project.)
```

Comment:
no space for "user-facing" (or should it be end-user facing?). "used in code" sounds weird;  can't think of a good alternative. Maybe remove that line altogether, since we already say end-users above? not sure